examples/visual_gen/visual_gen_wan_t2v.py

```
@@ -133,7 +133,25 @@ def parse_args():
         "Distributes sequence across GPUs for longer sequences. "
         "Requirements: num_heads (12) and sequence length must both be divisible by ulysses_size. "
         "Example: ulysses_size=2 on 4 GPUs with cfg_size=2 -> "
-        "2 CFG groups × 2 Ulysses ranks = 4 GPUs total.",
+        "2 CFG groups × 2 Ulysses ranks = 4 GPUs total. "
```

Ruff (0.15.9)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)